### PR TITLE
Update badger version. Potential fix for batch writes causing OOM Crashes in Application

### PR DESCRIPTION
**Pull request by DuncanSmith1126**
This is currently being tested - here in case anybody else wants to give it a shot.

### CODE REVIEW

File: pkg/sloop/queries/queryd3heatmap.go [hunk 1]
```diff
@@ -305,6 +305,21 @@ func eventCountsToOverlayMap(events map[typed.EventCountKey]*typed.ResourceEvent
 	return retMap, nil
 }
 
+func eventCountsToOverlayMapBucketed(events map[typed.EventCountKey]*typed.ResourceEventCounts, bucketWidth time.Duration) (map[typed.ResourceSummaryKey][]Overlay, error) {
```
Comment (thomashargrove) [inline pkg/sloop/queries/queryd3heatmap.go]: @DuncanSmith1126 this seems unrelated

Comment (DuncanSmith1126) [reply pkg/sloop/queries/queryd3heatmap.go]: Op - yup
